Cargo.toml

```diff
@@ -9,7 +9,9 @@ ledger_device_sdk = "1.0.0"
 ledger_device_ui_sdk = "1.1.0"
 ledger_secure_sdk_sys = "1.0.0"
 include_gif = "1.0.0"
-numtoa = "0.2.4"
+serde = {version="1.0.192", default_features = false, features = ["derive"]}
+serde-json-core = { git = "https://github.com/rust-embedded-community/serde-json-core"}
+
```

Comment:
@yogh333 it does not compile with 0.5.1 because of the `heapless` version it depends on. The master branch uses a newer version that compiles :shrug: